### ISSUE TITLE
Get strange error on webkit

### ISSUE THREAD

**Issue by Riey** [rustwasm/wasm-bindgen#2075]
### Describe the Bug

I'm not sure this is webkit issue or something

This example code success most time but often failed with strange error

### Steps to Reproduce

1. Clone https://github.com/Riey/yaml-error-test
2. Build with wasm-pack
3. Serve page with some server
4. Try it on webkit browser I've been tested in epiphany and webview(https://github.com/zserge/webview)
And show console for check error

### Expected Behavior

Get 36

![Screenshot from 2020-04-08 19-46-20](https://user-images.githubusercontent.com/14910534/78802441-ca3bfa00-79f8-11ea-8fd0-3168cb77b1b5.png)

### Actual Behavior

![Screenshot from 2020-04-08 19-46-41](https://user-images.githubusercontent.com/14910534/78802420-c5774600-79f8-11ea-91a4-1b16c0e32137.png)
![Screenshot from 2020-04-08 19-46-30](https://user-images.githubusercontent.com/14910534/78802436-c9a36380-79f8-11ea-8da8-a3fd4bbb2e55.png)
![Screenshot from 2020-04-09 00-27-30](https://user-images.githubusercontent.com/14910534/78802601-fbb4c580-79f8-11ea-9a91-ba72f2888777.png)

**Comment by alexcrichton** [comment]
Thanks for the report! I'd be curious to test this out locally, but it looks like your example repo is missing the JSON file?

**Comment by Riey** [comment]
@alexcrichton Thanks you I just add it sorry

**Comment by alexcrichton** [comment]
Thanks!

Can you confirm what wasm-bindgen/rustc versions you're using? I checked out your repo and built it and it works in Firefox/Chrome for me.

**Comment by Riey** [comment]
Yes this code work fine in firefox or chrome but not on epiphany

**Comment by Riey** [comment]
Well epiphany itself it not matter for me but I trying to browse rust wasm in webview(https://github.com/Boscop/web-view)

but it also fail since it use webkit for linux

**Comment by alexcrichton** [comment]
Oh sorry I see, but in that case this seems like it's likely a bug in the browser and/or wasm engine?

**Comment by Riey** [comment]
Then I'd like to report this in webkit bug report can I left this issue for others who like me?

**Comment by alexcrichton** [comment]
Sure!
